main/python-babel/template.py

```diff
@@ -13,9 +13,9 @@
 pkgdesc = "Tools for internationalizing Python applications"
 maintainer = "q66 <q66@chimera-linux.org>"
 license = "BSD-3-Clause"
-url = "http://babel.pocoo.org"
-source = f"$(PYPI_SITE)/B/Babel/Babel-{pkgver}.tar.gz"
-sha256 = "6919867db036398ba21eb5c7a0f6b28ab8cbc3ae7a73a44ebe34ae74a4e7d363"
+url = "https://babel.pocoo.org"
+source = f"$(PYPI_SITE)/B/Babel/babel-{pkgver}.tar.gz"
```

Comment:
i hope i got the url right, i'm not quite sure how these work

Reply:
is correct if it works